website/src/components/ComparisonTable.tsx

```diff
@@ -19,7 +19,7 @@ const workspaceRows: Comparison[] = [
 		feature: 'Written in',
 		support: {
 			moon: 'Rust',
-			nx: 'Node.js',
+			nx: 'Rust for speed & TypeScript for extensibility',
```

Comment:
@JamesHenry Do you have more on this? It seems like Nx only uses Rust for a very small portion, so this sentence seems disingenuous.

Reply:
This is about importance vs quantity.

Rust is very important to Nx but we are diligent about benchmarking, and only look to write things in Rust when it provides an objective, real-world speed benefit over TypeScript. In practice this is less often than people assume.

As per my previous comment Nx is architected in such a way that it has a language and framework agnostic core, and then receives that context specific knowledge via plugins.

It is only the Nx Core that is in scope for Rust, because having plugins in Rust has a high cost to the community. Writing plugins in TS/JS has a lower barrier to entry than Rust and is one of the reasons why it has proven to be so successful and why we receive so many community PRs to our first-party plugins. We have a lot of first-party plugins in our monorepo, so basing your comment around the number of files in one programming language vs another then it will appear heavily diluted.

Nx uses Rust for critical operations such as:
- project graph construction
- file system operations, including file watching
- globbing
- hashing
- new task runner which will be the default in the next version

These are the most important things that Nx does, and they are written in Rust. It is not disingenuous to say that Nx is written in Rust for speed and TypeScript for extensibility.

Hopefully that clarifies things!